README.md

```diff
@@ -81,6 +81,27 @@ Please note that for the moment, the Microsoft Azure endpoints can only be used
 For a detailed example on how to use fine-tuning and other operations using Azure endpoints, please check out the following Jupyter notebook:
 [Using Azure fine-tuning](https://github.com/openai/openai-python/blob/main/examples/azure/finetuning.ipynb)
 
+### Microsoft Azure Active Directory Authentication
+
+In order to use Microsoft Active Directory to authenticate to your Azure endpoint, you need to set the api_type to "azure_ad" and pass the acquired credential token to api_key. The rest of the parameters need to be set as specified in the previous section.
+
+
+```python
+from azure.identity import DefaultAzureCredential
```

Comment:
I think this needs to be added as a new dependency

Reply:
I don't think that the dependency is needed on our side, since only the user needs this dependency in order to create the authentication token. We just put this token into our request headers.

Reply:
Might need to change the way we do authentication: https://github.com/openai/openai-python/pull/92#discussion_r885453162

Reply:
We decided to leave it like this for now.